packages/dds/tree/docs/dataModel/README.md

```diff
@@ -0,0 +1,128 @@
+# Data Model
+
+## Introduction
+
+This document discusses how tree data is conceptually organized within the SharedTree DDS. It is primarily of interested to Fluid developers working on SharedTree.  Most SharedTree users will interact with tree data through a pre-existing API layer.  However, the data model may be of interest to advanced users who are implementing a specialized API (see #8989).
+
+The SharedTree data model is closely related to, but not precisely the same as:
+
+- The internal in-memory representation of tree data
+- The serialized tree data sent across the network
+- The format used to persist tree data in storage
+
+While each of the above involve specific encodings of the SharedTree data model, the SharedTree data model itself is not concerned with the byte-level representation.
+
+## Requirements
+
+### JSON Interoperability
+
+We take it as a given that the underlying data model for the SharedTree is tree-structured data. We also agree that JSON is the modern lingua franca of the web and services. We therefore begin with the following requirements:
+
+1. JSON must efficiently and losslessly round-trip to and from the underlying SharedTree data model, with some caveats.
```

Comment:
"with some caveats" should probably be more specific.

Reply:
Added 'Potential JSON Caveats' section, thx.